File: JavaScript/Iniciante/1190 - Área Direita.js
const O = String(lines.shift());

let matriz = Array();
let linha = Array();

for (let i = 0; i<12; i++) {
    for (let j = 0; j<12; j++) {
        let elemento = parseFloat(lines.shift());
        linha.push(elemento);
    }
    matriz.push(linha);
    linha = [];
}

let soma = 0;
let divisor = 0;
let limite = 10;

for (let m = 1; m<11; m++) {
    for (let n = 11; n>limite; n--) {
        soma += matriz[m][n];
        divisor++;
    }
    if (m < 5) {
        limite--;
    } else if (m > 5) {
        limite++;
    }
}

if (O === 'S') {
    console.log(soma.toFixed(1));
} else if (O === 'M') {
    let media = (soma/divisor).toFixed(1);
    console.log(media);
} else {

}

File: Python/Iniciante/1190 - Área Direita.py
O = str(input())

matriz = list()
linha = list()

for i in range(0, 12, 1):
    for j in range(0, 12, 1):
        elemento = float(input())
        linha.append(elemento)
    matriz.append(linha)
    linha = []

soma = 0
divisor = 0
limite = 10

for m in range(1, 11, 1):
    for n in range(11, limite, -1):
        soma += matriz[m][n]
        divisor += 1
    if m < 5:
        limite -= 1
    elif m > 5:
        limite += 1

if O == 'S':
    print("%0.1f" %(soma))
elif O == 'M':
    media = soma/divisor
    print("%0.1f" %(media))
